examples/nfs-pd/testpd.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1beta1
+desiredState:
+  manifest:
+    containers:
+      - name: testpd
+        image: dockerfile/nginx
+        volumeMounts:
+            # name must match the volume name below
+            - name: nfs
+              mountPath: "/var/www/html/mount-test"
```

Comment:
Perhaps that particular nginx image has issues?  I am running through volume examples on docker's website and I can set nginx's default webroot without the extra subdir.

See https://docs.docker.com/installation/mac/#mount-a-volume-on-the-container